docs/db/changelog/changesets/powerauth-java-server/1.4.x/20230322-init-db.xml

```diff
@@ -541,14 +541,16 @@
     </changeSet>
 
     <changeSet id="24" logicalFilePath="powerauth-java-server/1.4.x/20230322-init-db.xml" author="Lubos Racansky">
+        <validCheckSum>9:2e39673c03bcb1e72e12aeeaa54dc716</validCheckSum>
+        <validCheckSum>9:08e14a3059bf341524a549dc8bc9c1c3</validCheckSum>
         <preConditions onFail="MARK_RAN">
             <not>
                 <tableExists tableName="pa_operation_application"/>
             </not>
         </preConditions>
         <comment>Create a new table pa_operation_application</comment>
         <createTable tableName="pa_operation_application">
-            <column name="application_id" type="bigint">
```

Comment:
This will case some installation with bigint and some with int - problems for future increments etc. So no we will not do such a change

Reply:
Ok, reverted.